lib/api/src/ticket_administration.cpp

```diff
@@ -0,0 +1,160 @@
+#include "irods/ticket_administration.hpp"
+
+#undef rxTicketAdmin
+
+#ifdef IRODS_TICKET_ADMINISTRATION_ENABLE_SERVER_SIDE_API
+#    include "irods/rsTicketAdmin.hpp"
+#    define rxTicketAdmin rsTicketAdmin
+
+#else
+#    include "irods/ticketAdmin.h"
+#    define rxTicketAdmin rcTicketAdmin
+
+#endif // IRODS_TICKET_ADMINISTRATION_ENABLE_SERVER_SIDE_API
+
+#include "irods/irods_random.hpp"
+#include "irods/objInfo.h"
+#include "irods/rcConnect.h"
+#include "irods/rcMisc.h"
+#include "irods/rodsError.h"
+#include "irods/rodsKeyWdDef.h"
+#include "irods/irods_exception.hpp"
+#include "irods/irods_at_scope_exit.hpp"
+
+#include <charconv>
+#include <string>
+#include <sstream>
+
+namespace irods::experimental::administration::ticket
+{
+    /// 
+    /// \brief A function to make a new ticket name
+    /// 
+    /// \param[in] newTicket String memory address that stores the new ticket name
+    ///
+    void make_ticket_name(char* newTicket)
```

Comment:
Minor: Let's move this into an anonymous namespace outside of `irods::experimental::administration::ticket` since it is not used outside of this file.

Reply:
Still here